include/deal.II/base/types.h

```diff
@@ -130,6 +130,10 @@ namespace types
   typedef unsigned char material_id;
 }
 
+/**
+ * Declare type used in Epetra.
+ */
+typedef double TrilinosScalar;
 namespace TrilinosWrappers
```

Comment:
Let's add one or two empty lines between the typedef and the namespace, just to make it clear that they are entirely separate concerns.

In an ideal world, I would have moved the typedef inside the namespace, but that may or may not work any more since there was the previous `#define`...